### PR TITLE
Adding wms layers filters

### PR DESCRIPTION
**Pull request by smartsena**

### CODE REVIEW

File: support/admin.py [hunk 1]
```diff
@@ -156,4 +159,4 @@ class TmsLayerAdmin(admin.ModelAdmin):
 admin.site.register(models.TmsLayer, TmsLayerAdmin)
 admin.site.register(models.CategoryLayersGroup, CategoryLayersGroupAdmin)
 # admin.site.register(models.HeatmapLayer, HeatmapLayerAdmin)
```
Comment (dagnaldo) [inline support/admin.py:161]: Remove comments, please.

File: support/fixtures/support.yaml [hunk 2]
```diff
@@ -6160,3 +6160,52 @@
       queryable: false
       has_opacity: false
       default_opacity: null
+- model: support.layerfilter
```
Comment (dagnaldo) [inline support/fixtures/support.yaml:6163]: It have to be in an another fixure file.

File: support/serializers.py [hunk 3]
```diff
@@ -70,7 +70,7 @@ class Meta:
             'tms',
             'wms',
             # 'heatmap',
```
Comment (dagnaldo) [inline support/serializers.py:72]: Remove comments.

**Review by dagnaldo — CHANGES_REQUESTED** [review]
Check conflicts and my comments.

File: support/admin.py (hunk 1 above)
Comment (smartsena) [reply support/admin.py:161]: Removed.

File: support/serializers.py (hunk 3 above)
Comment (smartsena) [reply support/serializers.py:72]: Removed.

File: support/fixtures/support.yaml (hunk 2 above)
Comment (smartsena) [reply support/fixtures/support.yaml]: Created file support_layerfilter_fixtures.yaml.

**Review by dagnaldo — APPROVED** [review]
Well well, congratulations, sir.
you are improving your codes every day.
Approved!

**Pull request merged**